submissions/NastjonkaK/HTML-CSS-Popup/Style/style.css

```diff
@@ -0,0 +1,192 @@
+body {
+	font-family: "Montserrat";
```

Comment:
Will be good practice to add few web safe fonts
U can read this article http://codeguide.academy/html-css.html

Reply:
Thanks for additional information.